web/source/keymanweb.js

```diff
@@ -1711,13 +1711,13 @@
       e = window.event;
       if(!e)
       {
-        e = keymanweb._GetLastActiveElement();
-        if(e)
+        var e2 = keymanweb._GetLastActiveElement();
+        if(e2)
         {
-          e = e.ownerDocument;
```

Comment:
Google Closure didn't appreciate reuse of the variable in this case, since the involved types are different.